foursight_core/react/api/auth0_config.py

```diff
@@ -137,14 +137,16 @@ def get_client(self) -> str:
     def get_domain(self) -> str:
         return self.get_config_data()["domain"]
 
-    def get_secret(self) -> str:
+    @staticmethod
+    def get_secret() -> str:
         """
         Returns the Auth0 secret.
         Currently we get this environment variables setup from the GAC; see identity.py.
         """
         return os.environ.get("CLIENT_SECRET", os.environ.get("ENCODED_AUTH0_CLIENT"))
 
-    def get_callback_url(self, request: dict) -> str:
+    @staticmethod
```

Comment:
I assume rewrites like this are because some linter said it could be static. I almost always rewrite to a class method and that shuts them up, making me wonder why they don't say one or the other. The thing is, if it's static, I often wonder why put it in the class at all. Sometimes it makes sense. But if you're just saying "it doesn't care about the instances but it can still be in the class, I think it's better to just write it as a class method because then if you realize you need class variables, there's nothing to rewrite.

Not a big deal really. It's OK to leave it. But I wanted to remark on this in context.

Reply:
Yeah, I'm still a bit unclear on rules of thumb about this kind of thing, thanks for comments.